releasenotes/notes/deprecate-tracer-writer-b4921987ad47a832.yaml

```diff
@@ -0,0 +1,4 @@
+---
+deprecations:
+  - |
+    :py:attr:`Tracer.writer` is deprecated. To force flushing of buffered traces to the agent, use :py:meth:`Tracer.flush` instead.
```

Comment:
does this linking work as expected? do you need to include the module namespace for it as well? `ddtrace.tracer.Tracer.writer` ?